02_bubble/joel00wood/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"fmt"
+)
+
+func bubble(s []int) []int {
+	c := make([]int, len(s))
+	copy(c, s)
+	var swapped bool
+	for {
+		for i := 1; i < len(c); i++ {
+			if c[i-1] > c[i] {
+				c[i-1], c[i] = c[i], c[i-1]
+				swapped = true
+			}
+		}
+		if !swapped {
+			break
+		} else {
```

Comment:
you can move reseting swapped flag to top of the for loop.

please consider sth like:
```
a := []int{1, 2, 3, 4}
for i := 0; i < len(a); i++ {
        k := 3
        if a[i] >2 { k = 6}
        fmt.Println(k, a)
        }
```

Reply:
Updated as described